File: ex09/ex09_01/ex09_01.js
function main() {
  const persons = new Map([
    ["Chang", 45],
    ["Chen", 36],
    ["Yu", 39],
    ["Wang", 25]]);

  console.log(persons.get("Chang"));
  console.log(persons.get("Chen"));
  console.log(persons.get("Yu"));
  console.log(persons.get("Wang"));
  return 0;
}
main();

File: ex09/ex09_01/ex09_01.cpp
#include <iostream>
#include <map>
using namespace std;

int main() {
  map<string, int> persons = {
    {"Chang", 45},
    {"Chen", 36},
    {"Yu", 39},
    {"Wang", 25}
  };
  cout << persons["Chang"] << endl;
  cout << persons["Chen"] << endl;
  cout << persons["Yu"] << endl;
  cout << persons["Wang"] << endl;
  return 0;
}